components/layouts/quotes/quotes--single.hbs

```diff
@@ -0,0 +1,22 @@
+<div class="section--bg-white">
+  <div class="section__inner">
+    <div class="quote--single">
+
+      <picture class="quote__image">
+        <source srcset="https://via.placeholder.com/286x335" media="(min-width: 640px)">
+        <source srcset="https://via.placeholder.com/85x116" media="(min-width: 320px)">
+        <img src="https://" role="presentation" alt="">
+      </picture>
+
+      <blockquote class="quote__text">
+        <p>Time's up for talk—we need to take action.</p>
+      </blockquote>
+
+      <cite class="quote--attribution">
+        <span class="quote--name">John Chuang</span>
+        <span class="quote--title">Ceo, Aquent</span>
+      </cite>
```

Comment:
`<cite>` has to be a sibling of `<blockquote>` because it has to change position in mobile.
[Per spec](https://html.spec.whatwg.org/#the-cite-element) `<cite>` can have various parents.